lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -891,7 +891,13 @@ defmodule Kernel.ErrorsTest do
 
   test "invalid args for bodyless clause" do
     assert_compile_fail CompileError,
-      "nofile:2: can use only variables and \\\\ as arguments in definition header",
+      """
+      nofile:2: only variables and \\\\ are allowed as arguments in definition header.
+
+      If you did not intend to define a header, make sure your function definition has
```

Comment:
Actually, we need to change. The first sentece was changed too, anyways, I will change the test to match just the first line, it's better this way. Thanks.